### ISSUE TITLE
Evaluation Metrics

### ISSUE THREAD

**Issue by ArtemisiaW** [AnyiRao/SceneSeg#26]
Your paper uses 3 metrics, which are AP, Miou and Recall.

I wonder whether the best results of these three metrics are achieved by the same pretrained lgss model, or they are achieved by different pretrained models.

And can you provide the pretrained models?

**Comment by anyirao** [comment]
Hi Dear @ArtemisiaW, thanks for your interest. You may refer to this https://drive.google.com/open?id=1F-uqCKnhtSdQKcDUiL3dRcLOrAxHargz

**Comment by ArtemisiaW** [comment]
Hi @AnyiRao , thanks so much for the quick reply!

But there may be some ambiguity in my initial comment. What I mean are the models trained on MovieScene Dataset, which achieved the best experiment results. Can you provide these trained models?

Thanks a lot!

**Comment by ArtemisiaW** [comment]
Are the metris (recall, precision, miou) achieved by the same checkpoint with the same post-processing or do you select a best checkpoint for each metric?

**Comment by ArtemisiaW** [comment]
Are the metrics (recall, AP, miou) achieved by the same checkpoint with the same post-processing or do you select a best checkpoint for each metric?

**Comment by anyirao** [comment]
What the paper report is achieved by the same checkpoint. But I am sure that you could train different models to achieve the best for different metrics.

**Comment by ArtemisiaW** [comment]
OK, thank you very much! @AnyiRao
